### PR TITLE
Support for Watson SDK v7.0.0

### PR DESCRIPTION
**Pull request by tnixa**
### Summary
Updated the project for Watson SDK v7.0.0
https://github.com/watson-developer-cloud/java-sdk

- updated repositories and dependencies
- updated configurations and tests
- removed Watson Conversation as this has been replaced by Watson Assistant 
- added Watson Compare and Comply

- build was successful and all tests passed

### CODE REVIEW

File: src/test/java/com/ibm/watson/developer_cloud/spring/boot/test/CompareComplyAutoConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2017 IBM Corp. All rights reserved.
```
Comment (germanattanasio) [inline src/test/java/com/ibm/watson/developer_cloud/spring/boot/test/CompareComplyAutoConfigTest.java:2]: `Copyright © 2017` -> `Copyright © 2019`

**Review by germanattanasio — APPROVED** [review]
I just have one comment but besides that, it looks good.
I added @lpatino10 as a reviewer so that he can test the changes and once that's done we can merge this.

**Comment by germanattanasio** [comment]
We need to squash the commits with a `feat(...)` and probably make it a breaking change since the underlying service went up 2 major versions

**Review by mkistler — APPROVED** [review]
Looks good! 👍

**Comment by mkistler** [comment]
Agreed with @germanattanasio that we need to bump to a new major version.

**Review by lpatino10 — APPROVED** [review]
~Third the breaking change change. If you could just use bumpversion to upgrade to a new major version (see [here](https://github.ibm.com/igor-a-naumov/sso-w3id-saml-demo)), we can handle the release once this is merged.~

☝️ Actually, I can go ahead and do this after merging.

Otherwise, it looks good! Thanks a lot for your contribution 😄

**Pull request merged**